main/signers/lattice/Lattice/index.ts

```diff
@@ -301,7 +303,8 @@ export default class Lattice extends Signer {
       cb(null, addHexPrefix(signedTx.serialize().toString('hex')))
     } catch (err) {
       log.error('error signing transaction with Lattice', err)
-      return cb(new Error(err as string))
+      const latticeErrorMessage = (err as { errorMessage: string }).errorMessage
```

Comment:
looks good. can we create a Lattice-error specific type for this if they dont expose one?

Reply:
They don't expose one, I can add it

Reply:
https://github.com/GridPlus/gridplus-sdk/blob/1c1ba0712fe0317f4ed6d1ffcb82c4a5da239fa2/src/shared/errors.ts#L21